shared/test/test_helper.rb

```diff
@@ -39,6 +39,18 @@ def generate_library_for_test imports, protos
 end
 
 class ShowcaseTest < Minitest::Test
+  def new_client
```

Comment:
Would the name `echo_client` be more precise? Especially if we add tests for the non-Echo services in Showcase?

Reply:
Yeah, I suppose so. You mean helpers like `new_echo_client`, `new_echo_operations_client`, `new_identify_client`, etc.?

Reply:
Yep.

Reply:
Sure, I'll send a follow up shortly.